demos/powersync-supabase-yjs-text-collab-demo/.nvmrc

```diff
@@ -0,0 +1 @@
+v18.17.0
```

Comment:
It would be nice if this could use NodeJS 20 LTS (the same as in the root of the repo), is there maybe a reason for needing v18?

Reply:
Done